### PR TITLE
Coral-Trino: Improve readability of TrinoToRelConverterTest

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

**Comment by ebyhr** [comment]
@ljfgem Can you review this PR? I will close this PR if you think this isn't helpful.

File: coral-trino/src/test/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverterTest.java [hunk 1]
```diff
@@ -165,59 +165,73 @@ public Iterator<Object[]> getSupportedSql() {
...
             "SELECT \"foo\"()\n" + "FROM (VALUES  (0)) AS \"t\" (\"EXPR$0\")"))
-        .add(ImmutableList.of("select foo(10, 2)",
+        .add(new TrinoRelDataProvider("select foo(10, 2)",
             "LogicalProject(EXPR$0=[foo(+(*(10, 10), *(10, 2)))])\n" + "  LogicalValues(tuples=[[{ 0 }]])\n",
             "SELECT \"foo\"(10 * 10 + 10 * 2)\n" + "FROM (VALUES  (0)) AS \"t\" (\"ZERO\")"))
-        .build().stream().map(x -> new Object[] { x.get(0), x.get(1), x.get(2) }).iterator();
+        .build().stream().map(x -> new Object[] { x.trinoSql, x.expectedRelString, x.expectedSql }).iterator();
+  }
+
+  private static class TrinoRelDataProvider {
```
Comment (ljfgem) [inline coral-trino/src/test/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverterTest.java:224]: nit: `TrinoToRelTestDataProvider`?

**Review by ljfgem — COMMENTED** [review]
Thanks @ebyhr for this PR! I think it's helpful, will make SQLs much clearer in IntelliJ.
Left a nit.

File: coral-trino/src/test/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverterTest.java (hunk 1 above)
Comment (ebyhr) [reply coral-trino/src/test/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverterTest.java]: Thanks for your suggestion, renamed.

**Review by ljfgem — APPROVED** [review]
LGTM, thanks @ebyhr !

**Pull request merged**